roles/openshift_health_checker/test/elasticsearch_test.py

```diff
@@ -72,7 +72,7 @@ def test_check_elasticsearch():
     assert_error_in_list('NoRunningPods', excinfo.value)
 
     # canned oc responses to match so all the checks pass
-    def exec_oc(cmd, args):
+    def exec_oc(cmd, args, **_):
```

Comment:
Why is this?

Reply:
Updated some of the checks to add the `save_as_name` kwarg and don't really have an idiom for `lambda ignore-all-args-and-kwargs`